packages/component/package.json

```diff
@@ -44,7 +44,7 @@
     "react": "^16.3.0 || ^17.0.0 || ^18.0.0"
   },
   "dependencies": {
-    "@babel/runtime": "^7.7.7",
+    "@babel/runtime": "^7.12.18",
```

Comment:
7.12 is the first version that introduces `exports` field in package.json. We need it for ESM